pkg/features/antrea_features.go

```diff
@@ -155,7 +156,7 @@ var (
 		AntreaPolicy:                {Default: true, PreRelease: featuregate.Beta},
 		AntreaProxy:                 {Default: true, PreRelease: featuregate.Beta},
 		Egress:                      {Default: true, PreRelease: featuregate.Beta},
-		EndpointSlice:               {Default: true, PreRelease: featuregate.Beta},
+		EndpointSlice:               {Default: true, PreRelease: featuregate.GA, LockToDefault: true},
```

Comment:
I feel we can be more conservative and do not lock it to true since the code is able to handle Endpoint API anyway. This leaves a workaround when there is something wrong with EndpointSlice API.
It's necessary to lock a feature gate to true the code is changed to not support its false case, like no longer checking the feature gate when executing related code.

Reply:
If so, do we need to add an option to replace feature gate `EndpointSlice`?